x-pack/test/functional/es_archives/security_solution/README.md

```diff
@@ -9,3 +9,5 @@ or
 ```
 x-pack/test/api_integration/apis/security_solution
 ```
+
+* Folder `telemetry` is for the tests underneath `detection_engine_api_integration/security_and_spaces/tests/telemetry`.
```

Comment:
NOTE: This and all these e2e's were not part of the original PR but I added them from 8.0/8.1 as I was getting bad conflicts not having them and also I did not feel comfortable shipping 7.17.0 without having e2e tests.